pages/[league]/player/[id].tsx

```diff
@@ -0,0 +1,337 @@
+                  </TableWrapper>
+                </>
+              )}
+              <TableHeading>Ratings</TableHeading>
+              <TableWrapper>
+                <TableContainer>
+                  {!isSkater && (
+                    <SingleGoalieRatingsTable data={goalieRatings} />
+                  )}
+                  {isSkater && (
+                    <SinglePlayerRatingsTable data={skaterRatings} />
+                  )}
+                </TableContainer>
+              </TableWrapper>
+            </>
+          )
+          }
+        </Main >
+      </Container>
+      <Footer />
+    </React.Fragment >
+  );
+}
+
+export const getServerSideProps: GetServerSideProps = async (ctx) => {
+  try {
+    const { league, id } = ctx.query;
+
+    return { props: { league, id } };
+  } catch (error) {
+    ctx.res.statusCode = 404;
+
+    return { props: { error } };
+  }
+};
+
+const Main = styled.main`
+  height: 100%;
+  width: 100%;
+`;
+
+const TableWrapper = styled.div`
+  width: 95%;
+  margin: auto;
+`;
+
+const TableContainer = styled.div`
+  width: 100%;
+  margin: 30px 0;
+`;
+
+const TableHeading = styled.h2`
+  width: 95%;
+  margin: 30px auto;
+  font-size: 2.2rem;
+  padding: 5px 0;
+  border-bottom: 1px solid black;
+`;
+
+const SelectorWrapper = styled.div`
+  width: 250px;
+  float: right;
+`;
+
+const CenteredContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  margin-top: 10px;
+`;
+
+const ErrorBlock = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: ${({ theme }) => theme.colors.red200};
+  height: 50px;
+  padding: 10px;
+  margin: 10px 0;
+  font-weight: 500;
+`;
+
+const PlayerInfo = styled.div`
+  font-size: 1.1rem;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: bold;
+`;
+
+const Container = styled.div`
+  width: 75%;
+  padding: 1px 0 40px 0;
+  margin: 0 auto;
+  background-color: ${({ theme }) => theme.colors.grey100};
+
+  @media screen and (max-width: 1024px) {
+    width: 100%;
+    padding: 2.5%;
+  }
+`;
+
+const ButtonContent = styled.div`
+  display: flex;
+  flex-direction: row;
+  font-size: 14px;
+  font-weight: 700;
+`;
+
+const Button = styled.button`
+  width: 100%;
+  background-color: transparent;
+  padding: 6px 16px;
+  border: 1px solid
+    ${(props: StyleProps) => (props.inverse ? 'black' : 'white')};
+  color: ${(props: StyleProps) => (props.inverse ? 'black' : 'white')};
+  cursor: pointer;
+  border-radius: 5px;
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.blue600};
+  }
+
+  &:active {
+    background-color: ${({ theme }) => theme.colors.blue700};
+  }
+
+  @media screen and (max-width: 700px) {
+    padding: 6px 8px;
+  }
+`;
+
+const Table = styled.table`
+  width: 100%;
+`;
+
+const TableRow = styled.tr`
+`;
+
+const TableData = styled.td`
+  padding: 10px;
+`;
+
```

Comment:
yeah not really sure what the table is for in terms of styling 🤷

Reply:
these are corrected